server/views/pages/prisons/sessionTemplates/viewSingleSessionTemplate.njk

```diff
@@ -235,15 +235,13 @@
       </form>
 
       <h2 class="govuk-heading-m">Update this template</h2>
-      <form action="/prisons/{{ sessionTemplate.prisonId }}/session-templates/{{ sessionTemplate.reference }}/edit" data-test="template-edit-form" method="GET" novalidate>
```

Comment:
Unrelated to this PR but spotted that this button was a form. Shouldn't be as it is just a link and using it in a form was causing the CSRF value to be passed around as a URL parameter (because it's a `GET` form)